server/src/db/db-config.js

```diff
@@ -1,7 +1,15 @@
 import Sequelize from 'sequelize';
 
 //make path for production environment
-let databaseUrl = process.env.DATABASE_URL || 'postgres://localhost:5432/uncle';
+let databaseUrl;
+if (process.env.NODE_ENV = 'test') {
```

Comment:
are you setting the process.env.NODE_ENV to test here, and seeing if it exists, or are we flagging if the process.env.NODE_ENV is equal to test?